src/pages/Event.vue

```diff
@@ -1,11 +1,69 @@
 <template>
-  <div>Event</div>
+  <page-container>
+    <div :class="$style.titleContainer">
+      <page-title>{{ name }}</page-title>
+    </div>
+    <div :class="$style.infoContainer">
+      <p :class="$style.detail">{{ date }}</p>
+      <p :class="$style.detail">{{ place }}</p>
+      <p :class="$style.descirption">{{ descirption }}</p>
+    </div>
+    <hostname-list :hostnames="hostnames" />
+  </page-container>
 </template>
 
 <script lang="ts">
-import { defineComponent } from 'vue'
+import { defineComponent, computed, ref } from 'vue'
+import PageContainer from '/@/components/Layout/PageContainer.vue'
+import PageTitle from '/@/components/Layout/PageTitle.vue'
+import useParam from '/@/use/param'
+import HostnameList from '../components/Event/HostnameList.vue'
+import apis, { EventDetail } from '/@/lib/apis'
+import { getDisplayDuration } from '/@/lib/date'
+
+const getEvent = async (eventId: string): Promise<EventDetail> =>
+  (await apis.getEvent(eventId)).data
 
 export default defineComponent({
-  name: 'Event'
+  name: 'Event',
+  components: {
+    PageContainer,
+    PageTitle,
+    HostnameList
+  },
+  setup() {
+    const eventId = useParam('eventId')
+    const eventDetail = ref<EventDetail>()
+    getEvent(eventId.value).then(response => (eventDetail.value = response))
```

Comment:
```suggestion
    watchEffect(() => {
      getEvent(eventId.value).then(response => (eventDetail.value = response))
   })
```
今のところはないんですが、この実装だと`/events/A`から`/events/B`への遷移が起きた時に内容が更新されないバグがおそらく発生します
なのでこういう風に`eventId`が変わったときは再取得するようにしておいたほうがよいです 😉